isis/src/control/objs/BundleUtilities/BundleMeasure.h

```diff
@@ -94,6 +102,13 @@ namespace Isis {
                                                      bundle control measure **/
       QSharedPointer<BundleImage> m_parentBundleImage; /**< Parent image of this bundle control measure **/
       QSharedPointer<BundleObservation> m_parentObservation; /**< Parent bundle observation **/
+      
+      LinearAlgebra::Matrix *m_imagePartial;    /**< Image partials associated with the control measure **/
+      LinearAlgebra::Matrix *m_point3DPartial;  /**< Point partials associated with the control measure **/
+      LinearAlgebra::Matrix *m_targetPartial;   /**< Target partials associated with the control measure **/
```

Comment:
You need to add logic in the destructor to delete these if they have been initialized. See the logic I provided for setImagePartials and the proper way to clean these up.